docs/dsl_inspec.md

```diff
@@ -239,6 +239,43 @@ end
 
 Mixing this with other conditionals (like checking existence of the files etc.) can help to test different test paths using InSpec. This way you can skip certain tests which would 100% fail due to the way servers are prepared, but you know that the same test suites are reused later in different circumstances by different teams.
 
+Some notes about `only_if`:
+
+ * `only_if` applies to the entire `control`. If the results of the `only_if` block evaluate to false, the contents of the describe blocks will not be run. However, the describe block resources (`command('redis-cli SET test_inspec "HELLO"')` in the above example) that preceded the only_if WILL run. For this reason, `only_if` should usually come first in the control, prior to any `describe` blocks.
+
+```ruby
+control "don't do this" do
+  describe command("something-dangerous") do
+    # ...
+  end
+  # command("something-dangerous") already ran!!!
+  only_if { its_safe }
+end
+```
+
+Instead, do this:
+```ruby
+control "do this instead" do
+  only_if { its_safe }
+  # command("something-dangerous") won't run unless it is safe
+  describe command("something-dangerous") do
+    # ...
+  end
+end
+```
+
+ * `only_if` results in a skipped result for the control and all tests in the control, and is the only supported way of skipping an InSpec control. Other methods relying on RSpec-specific features such as `before` and `skip` may not work in the future.
```

Comment:
`skip` is used in our compliance content; primarily for compliance checks that are specified in a standard but  can't be automated.
We could suggest changing these to `only_if('This cannot be automated') { false }` but that's a lot less readable. 
I would lean towards considering `skip` supported and adding a test that proves that it works.

Reply:
Where is `skip` documented?

Reply:
OK, I tracked it down - it's an RSpec core feature, documented here - https://relishapp.com/rspec/rspec-core/docs/pending-and-skipped-examples/skip-examples#skipping-using-%60skip%60 . 

So, this PR is about `only_if`. I've removed the references to `skip`  (I had mentioned it because a commenter on #760 had asked specifically about it). If you feel documentation for `skip` should be added to InSpec, or even tests for it, that should probably be done on another issue / PR.